<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cTableExportDialog</class>
 <widget class="QDialog" name="cTableExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBoxWhat">
     <property name="title">
      <string>Exportálandó</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButtonName">
        <property name="text">
         <string>Csak a kijelölt rekod név/nevek</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonSelected">
        <property name="text">
         <string>A kijelölt sorok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonViewed">
        <property name="text">
         <string>Az egész megjelenített táblázat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonAll">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ebben az esetben a program újraolvassa a teljes adattáblát, az összes sort. A kijelölések elvesznek. Nagy adattáblák esetén ez hosszabb ideig is eltarthat. A művelet megszakítására nincs lehetőség.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>A teljes táblázat</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBoxTarget">
       <property name="title">
        <string>Cél</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButtonClip">
          <property name="text">
           <string>Vágólapra</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Fájlba</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonWin">
          <property name="text">
           <string>Ablakba</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxForm">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Formátum</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonCSV">
          <property name="text">
           <string>CSV</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonHTML">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>HTML</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFilePath">
       <property name="text">
        <string>Cél fájl :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFilePath">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFilePath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditFilePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonName</sender>
   <signal>clicked(bool)</signal>
   <receiver>radioButtonClip</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolButtonFilePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>cTableExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>cTableExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonName</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButtonFile</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonName</sender>
   <signal>toggled(bool)</signal>
   <receiver>radioButtonHTML</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonName</sender>
   <signal>clicked(bool)</signal>
   <receiver>radioButtonCSV</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
